batect.yml

```diff
@@ -72,7 +72,7 @@ tasks:
       A convenience task to build (but not run) ORT's functional tests.
     run:
       container: build
-      command: ./gradlew --stacktrace jar funTestClasses
+      command: ./gradlew --stacktrace -x :reporter-web-app:yarnBuild jar funTestClasses
```

Comment:
This needs to say `-x :plugins:reporters:web-app-template:yarnBuild` now. Same below.

Reply:
Right, copy paste error...